woocommerce/woocommerce-android
# [Google for Woo] Success handling

Closes: #11917

Despite the PR change size, about half of it is just Compose/Fragment code that is nearly identicaly to Blaze's success bottom sheet, so hopefully the code is not too big to review.



Please do not merge until PR target is `trunk`
### Description


This PR adds handling Google Ads campaign creation success. Once a Google Ads campaign is successfully created in the webview, the webview will close automatically and a bottom sheet showing successful message will be shown. This behavior applies to both campaign creation from My Store card, and from More Menu:

| My Store | More Menu |
|-|-|
| ![success-my-store](https://github.com/user-attachments/assets/1591a539-b594-4e97-aad7-20dc00bb14ef) | ![success-more-menu](https://github.com/user-attachments/assets/ee0ff44e-4b09-4a88-b61a-a1adc32e4aa4) |

# Testing instruction:

Testing instruction
1. Pre-requisite: Install, activate, and set up the Google Ads plugin to a test site following pe5sF9-2Qo-p2. Ensure that you're using the plugin version 2.7.5 or above,
2. Run app,
3. Go to more menu and tap "Google for WooCommerce"
4. Depending on whether there are existing campaigns or not, it will open either campaign creation or dashboard. In the latter case, scroll until you find "Add paid campaign" button and tap it,
5. Complete the campaign creation steps,
6. The webview should close automatically and the success bottom sheet like in the screenshot above should appear,
7. Close it,
8. Go to My Store, enable Google Ads campaigns card if you haven't.
9. Try the campaign creation step from the card and complete it,
10. The webview should close automatically and the success bottom sheet like in the screenshot above should appear,
11. Close it.
